### DISCUSSION TITLE
Preserving API for reporting

### DISCUSSION THREAD

**Discussion by Firecul** [ThioJoe/YT-Spammer-Purge#1000] [Ideas]
Currently the program seems to retrieve profile pictures first (when that feature is enabled) then it will report the comments. 
I think it might be better to have the reporting happen first. If the API calls are going to run out I'd rather be able to report more comments than retrieve all the pictures.

![WindowsTerminal_Windows_PowerShell (84)](https://user-images.githubusercontent.com/34950371/210122080-3dfa99d3-5fbf-425a-91ab-2712e2264e10.png)

**Comment by Firecul** [comment]
Retrieving the pictures can be interesting. When you look at the ids you can sometimes make out bots that have pivoted from impersonating one creator to another for example.

**Comment by ThioJoe** [comment]
Hm not sure I'd really make this a priority tbh. The profile pictures thing is basically just in case someone wants as much data as possible to do machine learning or something, so I imagine the number of users of that feature is extremely low

**Reply by Firecul** [reply]
Almost certainly it's not used by many I agree. It's just an idea I had after noticing the apparent order of operations.
